Makes/ViewController.h

```diff
@@ -0,0 +1,14 @@
+//
+//  ViewController.h
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/8/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ViewController : UIViewController <UICollectionViewDataSource,UICollectionViewDelegateFlowLayout>
```

Comment:
Your class name should be MakesViewController or MakesCollectionViewController not only view controller